src/status_im/ui/components/checkbox/view.cljs

```diff
@@ -1,11 +1,16 @@
 (ns status-im.ui.components.checkbox.view
   (:require [status-im.ui.components.checkbox.styles :as styles]
-            [status-im.ui.components.react :as react]))
+            [status-im.ui.components.react :as react]
+            [status-im.utils.platform :as platform]))
 
-;; TODO(jeluard) Migrate to native checkbox provided by RN 0.49
-;; https://facebook.github.io/react-native/docs/checkbox.html
+(defmulti checkbox #(keyword platform/platform))
 
-(defn checkbox [{:keys [on-value-change checked?]}]
+(defmethod checkbox :android [{:keys [on-value-change checked?]}]
```

Comment:
How about using a plain `if` here? Platform set is closed and if a new update of RN supports more platform we will reflect this here.

Reply:
@jeluard yes, that makes sense.